doc/source/conf.py

```diff
@@ -649,7 +685,23 @@ def process_class_docstrings(app, what, name, obj, options, lines):
 ]
 
 
+def rstjinja(app, docname, source):
+    """
+    Render our pages as a jinja template for fancy templating goodness.
+    """
+    # http://ericholscher.com/blog/2016/jul/25/integrating-jinja-rst-sphinx/
+    # Make sure we're outputting HTML
+    if app.builder.format != 'html':
```

Comment:
Hmm I should look into what happens for PDF. We should be able to handle it just fine.